src/components/experiment-tracking/time-series/time-series.test.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { TimeSeries } from './time-series';
+import { HoverStateContext } from '../utils/hover-state-context';
+import { data } from '../mock-data';
+
+const selectedRuns = [
```

Comment:
if you merge the other main branch in, I already have this mock in the mock-data, you can just re-use from it